docs/sections/user_guide/cli/tools/rocoto/realize-exec-file.out

```diff
@@ -1,5 +1,5 @@
-[2024-08-02T00:43:07]     INFO 0 UW schema-validation errors found
-[2024-08-02T00:43:07]     INFO 0 Rocoto validation errors found
+[2024-08-26T23:11:41]     INFO 0 UW schema-validation errors found in Rocoto config
+[2024-08-26T23:11:41]     INFO 0 Rocoto XML validation errors found
```

Comment:
Added `XML` to this message to make it extra clear that it's not the UW YAML config that was validated on the previous line.